### ISSUE TITLE
Did you succeed to train and test the model?

### ISSUE THREAD

**Issue by wjkim1103** [moabitcoin/Adversarial-video-summarization-pytorch#3]
Hello, 
thanks for sharing your code. It seems good to have the pre-trained LSTM autoencoder model  in your code.
I tried to train this model using different code,  but it is so difficult to train and test the model. 
how about your result? Are the losses converged? If possible, can you upload the results of your experiment?
And in the s_p part, it seems to multiply every element of the feature by a uniform sample. Shouldn't s_p have one scalar value for each frame?

Thank you

**Comment by rebornwwp** [comment]
I have the same problems? I think there are many bugs in the code.  
Hi How do you run your code with lstmGan
Thanks very much

**Comment by wjkim1103** [comment]
> I have the same problems? I think there are many bugs in the code.
> Hi How do you run your code with lstmGan
> Thanks very much

I tried to use @j-min's code.
Did you try to run moabitcoin's code? there are many bugs in this code?

**Comment by rebornwwp** [comment]
> 
> 
> > I have the same problems? I think there are many bugs in the code.
> > Hi How do you run your code with lstmGan
> > Thanks very much
> 
> I tried to use @j-min's code.
> Did you try to moabitcoin's code? there are many bugs in this code?

Yes, I read the code. follow the description to run the code, but some bugs make the code to run with exception.
